docs/index.md

```diff
@@ -9,31 +9,45 @@ These dbt-core features include:
+1. add a new version to `fct_orders`
+
+and that's it!
 
-Each of the available [commands page](commands.md) allows you to add one (or many) of the above features to a set of models specified by the selection syntax in the command.
+The **global** commands combine _multiple_ **component** commands to complete a larger set of work and begin with `dbt-meshify`. For example, if you wanted to define a group for a subset of your models, you would run something like `dbt-meshify group finance --owner-name "Monopoly Man" --select +tag:finance`. This command would:
+1. define a new group named "finance" in your dbt project, setting the owner name to "Monopoly Man"
+2. add all models tagged with "finance" to that new group
+3. set `access` to protected for all "leaf" models (models with no downstream dependencies) and models with cross-group dependencies
+4. add contracts to all public nodes ... TO DO: if access is "protected" instead of "public" should we add contracts? would need to update examples below as well
 
-The goal of this package is to make it more straightforward to apply to your project so that splitting apart a monolithic project into component projects is a more automated, dbt-tonic experience.
+all at once!
 
-The process of splitting a dbt monolith apart roughly requires you to:
+The next question to ask yourself is "which of my models do I want to add these feature(s) to?". This informs the selection syntax you provide to the `dbt-meshify` command of choice. `dbt-meshify` uses the same selection syntax as `dbt`, so you can use the `--select`, `--exclude` and `--selector` flags to select resources based on model names, tags, and so on!
 
-1. Determine what parts of your project should be grouped together into subprojects
-2. Determine the access-level for the members of that group
-3. Add model contracts to the elements that are public and accessed my members outside the group specified in (1)
-4. (Optional) Add model versions to the public models to allow for development without impacting downstream stakeholders.
+Once you've decided:
+1. which governance feature(s) you want to add to your dbt project
+2. which subset of models you want to add those feature(s) to
 
-Here's how that might look for the process of creating a separate `finance` subproject in your dbt monolith.
+you're ready to use `dbt-meshify`!
 
-```bash
-# create a group of all models tagged with "finance"
-# leaf nodes and nodes with cross-group dependencies will be `public`
-# public nodes will also have contracts added to them
-dbt-meshify group finance --owner-name "Monopoly Man" -s +tag:finance
+For further information, check out the available [commands](commands.md) or read through some [examples](examples.md).
 
-# optionally use the add-version operation to add a new version to a model
-dbt-meshify operation add-version -s fct_orders
-```
+## What dbt-meshify does not handle
```

Comment:
@dave-connors-3 @nicholasyager would love to get your thoughts on if anything else should be called out here!

Reply:
🤔 Some low risk thoughts
- `split` doesn't copy over all project files, so there may be documentation that's not copied over (like markdown files).
  - Now that I'm thinking about it, does split copy over files with docblocks?
- `group` and `create-group` will update existing groups already defined in yml.

Reply:
What would be the manual checks for these?
- look through markdown files and see if there's any documentation you want to manually bring over (opened a new issue to check about doc blocks https://github.com/dbt-labs/dbt-meshify/issues/158)
- ??